src/lib/seam/components/CreateClimateSettingScheduleForm/CreateClimateSettingScheduleForm.tsx

```diff
@@ -0,0 +1,63 @@
+import { createIsoDate } from 'lib/dates.js'
+import { useCreateClimateSettingSchedule } from 'lib/seam/thermostats/climate-setting-schedules/use-create-climate-setting-schedule.js'
+import {
+  ClimateSettingScheduleForm,
+  type ClimateSettingScheduleFormSubmitData,
+} from 'lib/ui/ClimateSettingForm/ClimateSettingScheduleForm.js'
+
+import type { CommonProps } from '../common-props.js'
+
+export type CreateClimateSettingScheduleFormProps = CommonProps
+
+export function CreateClimateSettingScheduleForm({
+  className,
+  onBack,
+}: CreateClimateSettingScheduleFormProps): JSX.Element | null {
+  return <Content className={className} onBack={onBack} />
+}
+
+function Content({
+  className,
+  onBack,
+}: CreateClimateSettingScheduleFormProps): JSX.Element {
+  const { submit, isSubmitting } = useSubmitCreateClimateSettingSchedule(onBack)
+
+  return (
+    <ClimateSettingScheduleForm
+      className={className}
+      onBack={onBack}
+      onSubmit={submit}
+      isSubmitting={isSubmitting}
+    />
+  )
+}
```

Comment:
I think we can combine.

```suggestion
  const { submit, isSubmitting } = useSubmitCreateClimateSettingSchedule(onBack)

  return (
    <ClimateSettingScheduleForm
      className={className}
      onBack={onBack}
      onSubmit={submit}
      isSubmitting={isSubmitting}
    />
  )
}
```

Reply:
[3c6d378](https://github.com/seamapi/react/pull/440/commits/3c6d378a011411fd70e50a68e3a80584d0880bf7)